mongox/models.py

```diff
@@ -336,6 +336,23 @@ async def insert(self: T) -> T:
         self.id = result.inserted_id
         return self
 
+    @classmethod
+    async def insert_one(cls: typing.Type[T], model: T) -> T:
```

Comment:
To be honest I'm not a huge fan of this one. The final result will be:

```python
movie = Movie()
await Movie.insert_one(movie)
```

Though this is simpler:

```python
await Movie().insert()

# Or
movie = Movie()
await movie.insert()
```

Is there any reason for that?

Reply:
`instance.insert()`	provide a convenient way to insert data, use `insert_one` just want to keep behavior consistent with `insert_many()`,  they all belong to the class method.

Maybe use `instance.insert` is enough.

Reply:
Maybe we should provide `Movie.insert(movie)` method 😁

Reply:
I don't know. I would try to keep this as small as possible. Having to deal with multiple ways of doing the same thing can be challenging 😄

Reply:
By the way, this needs to be documented too. Looks great now.